app/models/miq_server/worker_management/monitor/stop.rb

```diff
@@ -14,12 +14,8 @@ def stop_worker_queue(worker, monitor_reason = nil)
   end
 
   def stop_worker(worker, monitor_reason = nil)
-    w = worker.kind_of?(Integer) ? miq_workers.find_by(:id => worker) : worker
-
-    if w.nil?
-      _log.warn("Cannot find Worker <#{w.inspect}>")
```

Comment:
Noticed this was removed.  Any particular reason or is it just not useful?

Reply:
This log entry was pulled out into `find_worker`.
I wanted to reuse this when I transitioned 'kill_async` over to be more similar to `stop_worker_queue`.

Maybe the log should not be hidden in a finder method and I should drop this commit